x-pack/legacy/plugins/cross_cluster_replication/public/app/sections/home/auto_follow_pattern_list/components/auto_follow_pattern_table/auto_follow_pattern_table.js

```diff
@@ -134,6 +136,19 @@ export class AutoFollowPatternTable extends PureComponent {
         ),
         sortable: true,
       },
+      {
+        field: 'active',
+        dataType: 'boolean',
+        align: 'center',
+        name: i18n.translate(
+          'xpack.crossClusterReplication.autoFollowPatternList.table.activeTitle',
+          {
+            defaultMessage: 'Active',
+          }
+        ),
+        render: active => <EuiHealth color={active ? 'success' : 'danger'} />,
```

Comment:
I noticed for follower indices, we also include the status text.

<img width="143" alt="Screen Shot 2020-02-04 at 3 04 37 PM" src="https://user-images.githubusercontent.com/5226211/73782855-be198f00-4760-11ea-8f20-4983b82223e4.png">